hack/testdata/levee/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- spiffxp
+reviewers:
+- spiffxp
```

Comment:
I'll accept this for now, when we update with other members, I would like to be removed

Reply:
Thanks!